src/java/org/apache/cassandra/stress/settings/SettingsSchema.java

```diff
@@ -93,6 +96,13 @@ public void createKeySpacesNative(StressSettings settings)
             //Add standard1
             client.execute(createStandard1StatementCQL3(settings), org.apache.cassandra.db.ConsistencyLevel.LOCAL_QUORUM);
 
+
+            if (cmd_type == Command.COUNTER_WRITE)
```

Comment:
Shouldn't it be mutually exclusive?
Create either counter or another table?

Reply:
is this still valid?

Reply:
It is valid request, I would like to see it be done for a single table - the one which is used.
Just not critical, critical to have schema creation for counters.